api/linters.yaml

```diff
@@ -94,11 +94,28 @@ components:
         errors:
           type: array
           items:
-            type: string
+            $ref: "#/components/schemas/LinterResultPluginRuleResultError"
         passed:
           type: boolean
         severity:
           type: string
           enum:
             - error
             - warning
+    LinterResultPluginRuleResultError:
+      type: object
+      properties:
+        error:
+          type: string
+        value:
+          type: string
+        expected:
+          type: string
+        level:
+          type: string
+        description:
+          type: string
```

Comment:
Should we include a field for the documentation URL? or is it your idea that it could be part of the `suggestions` field?

Reply:
I think we can decide that later. Now that we have the `Error` struct we can increment the fields without breaking the compatibility.